### ISSUE TITLE
Increase Spawns!

### ISSUE THREAD

**Issue by lilgandhi1199** [TheDragonTeam/MobSettings#5]
Hello!
I am just trying to radically increase the mob spawns and I have used "INCREASEMOBS" mods but I think that yours will be more stable and less laggy since it actually is using an established library.

Thank You. Am I doing something wrong?

There is my cfg file

https://pastebin.com/raw/CGzRQMTK

and there is my crash log

https://pastebin.com/raw/Fjp7PgXn

**Comment by sokratis12GR** [comment]
Hello, thanks for your report. Will look into that issue soon.

**Comment by lilgandhi1199** [comment]
Thank you so much! It might just be too high of a value? I'll test that soon as well...

**Comment by sokratis12GR** [comment]
Oh, I just saw it... the values for max/min spawn need to be listed seperately for each biome, its
list of biomes:
- a
- b
- c

list of max spawns:
- 10
- 10
- 11

list of min spawns:
- 2
- 3
- 2

**Comment by lilgandhi1199** [comment]
The biomes ARE all listed seperately in my cfg (do I need the bullet points) Also the max or min is just 1 number (it's a hard limit...) so I don't understand why you have three.
Wait are you saying it should look like this?

"minecraft:ocean:1000:10"

Maybe you can just do it right for one monster and I'll copy paste for the rest of entities?

**Comment by lilgandhi1199** [comment]
I did what I think you mean. Is this right?
https://pastebin.com/raw/ZCF3eSst

**Comment by sokratis12GR** [comment]
No, uhm I ment that for each listed biome you gotta add a new line with spawn min and max like:
https://gist.github.com/5fc7359e96f604e33d58097742fcff31

**Comment by lilgandhi1199** [comment]
Awesome! Thank You
Btw I wonder if you have any interest in writing a SPRINTbar mod with me which uses the hunger bar?
You seem like you enjoy a challenge! and I don't know forge (only knew a little of bukkit awhile back)

It's just like battlefield where you bar drains and then you have to walk while you wait for it to rise up again **!or!** you can eat to make it rise faster (which is a behavior you'd just leave untouched in vanilla ofc). Think about mushroom soup which would be the best because you wouldn't even have to slow down and chomp on food!

I think it would be easy to code because you wouldn't have to modify existing vanilla behavior because you don't even have to check wither you were running or doing anything that modifies it's current value. You just are checking to see if it's below max hunger then raise current value by x per second. The config file would get to determine X and that's it.

If not can you point me in the right direction or any starting dangers or tips?
Thank You Very Much

**Comment by lilgandhi1199** [comment]
If that does not interest you and you can't help me I will close the issue so it doesn't look unattended.
Let me know Thank You!

**Comment by sokratis12GR** [comment]
Yeah, thanks for the invite, but I'm currently unable to help you, I just don't have enough time for more things. good luck tho!

**Comment by lilgandhi1199** [comment]
Take care

**Comment by lilgandhi1199** [comment]
So you didn't put enough maxs and mins for the number of biomes and I fixed that but it still crashes.

Just in case here is the new crash log
https://pastebin.com/raw/jjrArbzv
and the cfg
https://pastebin.com/raw/DM3qFPD1

and it says the array is out of bounds so that sounds like it's too high. But how high is too high?

**Comment by sokratis12GR** [comment]
hmm try this one out, https://gist.github.com/sokratis12GR/3fb19f66cd67d30e1a7851003cf2edf1 they gotta be the exact same amount but if this doesn't work I'll invastigate later for the bug.

**Comment by lilgandhi1199** [comment]
I generated a fresh cfg file and just replaced your crepper with its.
Here's the new crash log https://pastebin.com/raw/2SYH7BuY

**Comment by sokratis12GR** [comment]
hmm, interesting, definatelly a bug then

**Comment by lilgandhi1199** [comment]
I replaced 1000 with 20 and it still crashed
crash log appears the same
https://pastebin.com/raw/3zTM0uf7

**Comment by lilgandhi1199** [comment]
I replaced 10 with 1 and same thing

**Comment by lilgandhi1199** [comment]
Jesus by process of elimination it seems to me that your mod is just broke...
here's the cfg with ONLY 3 ENTRIES
https://pastebin.com/raw/3JSmMGhp

**Comment by lilgandhi1199** [comment]
It could be those other libraries too. I'll downgrade and see if I can find the version it worked for you

**Comment by lilgandhi1199** [comment]
NVM...
there is in fact no older 1.12.2 versions of either of those mods. They've yet to update them so I don't understand how your mod functioned at all let alone enough to give you the confidence to release. What am I missing?
thedragonlib-1.12.2-5.2.0.jar
Forgelin-1.5.1.jar

Edit, Tried
Forgelin-1.5.0.jar which is for 1.12

Still crashed
